packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -39,6 +48,8 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 ## `3.2.0`
 
+- BugFix: Resolved a bug where extenders adding `ssh` or `base` to `allTypes` on `ProfileCache` will result in duplicate nodes when adding a profile of that type in Zowe Explorer. [#3625](https://github.com/zowe/zowe-explorer-vscode/pull/3625)
```

Comment:
Looks like this was moved out of the bug fixes section - it can be kept on under "Bug fixes" for 3.2.0